phpstan-baseline.neon

```diff
@@ -0,0 +1,102 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Method Elastica\\\\QueryBuilder\\:\\:aggregation\\(\\) should return Elastica\\\\QueryBuilder\\\\DSL\\\\Aggregation but returns Elastica\\\\QueryBuilder\\\\Facade\\.$#"
+			count: 1
+			path: src/QueryBuilder.php
+
+		-
+			message: "#^Method Elastica\\\\QueryBuilder\\:\\:collapse\\(\\) should return Elastica\\\\QueryBuilder\\\\DSL\\\\Collapse but returns Elastica\\\\QueryBuilder\\\\Facade\\.$#"
+			count: 1
+			path: src/QueryBuilder.php
+
+		-
+			message: "#^Method Elastica\\\\QueryBuilder\\:\\:query\\(\\) should return Elastica\\\\QueryBuilder\\\\DSL\\\\Query but returns Elastica\\\\QueryBuilder\\\\Facade\\.$#"
+			count: 1
+			path: src/QueryBuilder.php
+
+		-
+			message: "#^Method Elastica\\\\QueryBuilder\\:\\:suggest\\(\\) should return Elastica\\\\QueryBuilder\\\\DSL\\\\Suggest but returns Elastica\\\\QueryBuilder\\\\Facade\\.$#"
+			count: 1
+			path: src/QueryBuilder.php
+
+		-
+			message: "#^Function GuzzleHttp\\\\Psr7\\\\stream_for not found\\.$#"
```

Comment:
Is this caused by a missing library, or a Guzzle version mismatch?

Reply:
because version mismatch, `stream_for` does not exists in the installed version (I haven't check if the call could be removed, I'll try later)

Reply:
Checking this, we only have `guzzlehttp/guzzle` in `require-dev`, we don't have any `conflict` restricting versions used, so right now I think it should be fine ignoring the error, I'll open an issue about this.

Reply:
For reference: #2066